robotbase/src/main/java/frc/robot/Robot.java

```diff
@@ -102,7 +104,9 @@ public void robotPeriodic() {
 
   /** This function is called once each time the robot enters Disabled mode. */
   @Override
-  public void disabledInit() {}
+  public void disabledInit() {
+    new WaitCommand(3).andThen(new DriveCoast()).schedule();
```

Comment:
Make a member variable `m_setCoastOnDisable` which initializes to `new WaitCommand(Contants.Swerve.TIME_TO_COAST).andThen(new DriveSetCoast())`

In disabled init: `m_setCoastOnDisable.schedule()`

In teleopInit: `m_setCoastOnDisable.cancel()`

In Constants.SWERVE class make ` public static final Time TIME_TO_COAST= Units.Seconds.of(3);`